### PR TITLE
fix: Same CTA description 

### PR DESCRIPTION
**Pull request by BlueAcidFrog**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: src/page-content/content-cloud.tsx [hunk 1]
```diff
@@ -31,18 +31,18 @@ export const hero_alt: HeroImageProps = {
   id: 'cloud.hero',
   title: (
     <>
-      <span className={'is-themed-primary'}> Cloud native </span> identity
+      <span className={'is-themed-primary'}> Ory Cloud </span> uses idustry standards
```
Comment (vinckr) [inline src/page-content/content-cloud.tsx:34]:
```suggestion
      <span className={'is-themed-primary'}> Ory Cloud </span> uses industry standards
```

File: src/page-content/content-cloud.tsx [hunk 2]
```diff
@@ -31,18 +31,18 @@ export const hero_alt: HeroImageProps = {
...
     </>
   ),
   description: (
     <>
-      Integrate modern identity services with Ory Cloud. Batteries included.
-      <span className={'is-themed-primary'}> Based on Ory Open Source. </span>
+      Ory Cloud is API-first, customizable, privacy conform and secure and deploys all Ory services in concert - at any scale, across the globe.
```
Comment (vinckr) [inline src/page-content/content-cloud.tsx:39]:
```suggestion
      Ory Cloud is API-first, customizable, privacy conform, secure, and deploys all Ory services - at any scale, across the globe.
```

not sure what is meant by `in concert`

File: src/page-content/content-cloud.tsx [hunk 3]
```diff
@@ -67,7 +67,7 @@ export const projectListCloud: ProjectListProps = {
   description: (
     <>
       Ory Cloud works on any scale and stack and provides SDKs for any
-      programming language.
+      programming language.
```
Comment (vinckr) [inline src/page-content/content-cloud.tsx:70]:
```suggestion
      programming language.
```

File: src/page-content/navigation/header.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
       "mainMenu": [
         {
           "title": "Ory Cloud",
-          "description": "Start Free, No Credit Card required",
+          "description": "Start Free",
```
Comment (vinckr) [inline src/page-content/navigation/header.json:8]:
```suggestion
          "description": "Start Free",
```

you know my opinion on this.
keep  "No credit card required" 😁

File: src/page-content/navigation/header.json [hunk 5]
```diff
@@ -25,11 +25,11 @@
       "mainMenu": [
         {
           "title": "Ory Open Source",
-          "description": "Ory builds free and open software",
+          "description": "Ory builds open software",
```
Comment (vinckr) [inline src/page-content/navigation/header.json:28]:
```suggestion
          "description": "Ory builds open software",
```

free as in freedom not "for free" ! 
people want to pay us, they just want the freedom to modify things

File: src/page-content/navigation/header.json [hunk 6]
```diff
@@ -61,22 +61,22 @@
       "mainMenu": [
         {
           "title": "Documentation",
-          "description": "Documentation for all Ory products",
+          "description": "Open Source Documentation",
```
Comment (vinckr) [inline src/page-content/navigation/header.json:64]:
```suggestion
          "description": "Open Source Documentation",
```

This is also the documentation for Ory Cloud! 
Revert this as it is misleading IMO.

File: src/page-content/navigation/header.json [hunk 7]
```diff
@@ -61,22 +61,22 @@
...
           "to": "/docs"
         },
         {
           "title": "Community",
-          "description": "Be part of the Ory Community",
+          "description": "Be part of Ory Community",
```
Comment (vinckr) [inline src/page-content/navigation/header.json:69]:
```suggestion
          "description": "Be part of the Ory Community",
```

"Sei ein Teil der Gemeinschaft"
vs.
"Sei ein Teil Gemeinschaft"

🤔

File: src/page-content/navigation/navigation-announcement.tsx [hunk 8]
```diff
@@ -3,9 +3,9 @@ import React from 'react'
 import { PropTypes as AnnouncementProps } from '../../components/layouts/navigation/announcement'
 
 const jobs: AnnouncementProps = {
-  longText: 'Ory is hiring - shape the future of identity with us!',
-  shortText: 'Ory is hiring!',
-  link: '/jobs?mtm_campaign=job-banner'
+  longText: 'Get Ory Cloud Start Up subscription free for a year using code FIRST900',
+  shortText: 'Ory Cloud Start Up Plan!',
```
Comment (vinckr) [inline src/page-content/navigation/navigation-announcement.tsx:7]:
```suggestion
  shortText: 'Ory Cloud free for a year!',
```

sounds more exciting ;-)

File: src/page-content/navigation/header.json [hunk 9]
```diff
@@ -39,7 +39,7 @@
           "to": "/hydra"
         },
         {
-          "title": "Global Authorization Server",
+          "title": "Global Authorization",
```
Comment (vinckr) [inline src/page-content/navigation/header.json:42]:
```suggestion
          "title": "Global Authorization Service",
```

What do you think of this? was patricks suggestion.

"Global Authorization" is a bit vague, that could mean anything from KYC to bodyguard services.

**Review by vinckr — CHANGES_REQUESTED** [review]
I left some comments. 
In general this looks good, a few things have to be fixed.

File: src/page-content/navigation/header.json (hunk 5 above)
Comment (BlueAcidFrog) [reply src/page-content/navigation/header.json:28]:
```suggestion
          "description": "Ory builds free and open software",
```

**Review by vinckr — CHANGES_REQUESTED** [review]
This has some conflicts, it is probably if you copy the changes from here and make a fresh PR @BlueAcidFrog.

If this is not meant to be merged, please close.

**Comment by BlueAcidFrog** [comment]
This can be merged! @vinckr

File: src/page-content/navigation/navigation-announcement.tsx [hunk 10]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 
 import { PropTypes as AnnouncementProps } from '../../components/layouts/navigation/announcement'
 
-const jobs: AnnouncementProps = {
+const jobs: AnnouncementProps =
```
Comment (vinckr) [inline src/page-content/navigation/navigation-announcement.tsx:5]:
```suggestion
const jobs: AnnouncementProps = {
```

**Review by vinckr — APPROVED** [review]

**Pull request merged**